docs/docs/guides/labs/components/integrations/sling-component-tutorial.md

```diff
@@ -0,0 +1,75 @@
+---
+title: 'Dagster & Sling with components'
+description: The dagster-sling library provides a SlingReplicationCollectionComponent, which can be used to represent a collection of Sling replications as assets in Dagster.
+sidebar_position: 200
+---
+
+import DgComponentsPreview from '@site/docs/partials/_DgComponentsPreview.md';
+
+<DgComponentsPreview />
+
+The [dagster-sling](/integrations/libraries/sling) library provides a `SlingReplicationCollectionComponent` which can be used to easily represent a collection of [Sling](https://slingdata.io/) replications as assets in Dagster.
+
+## 1. Prepare a Dagster project
+
+To begin, you'll need a Dagster project. You can use an [existing components-ready project](/guides/labs/dg/incrementally-adopting-dg/migrating-project) or create a new one:
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/integrations/sling-component/1-scaffold-project.txt" />
+
+Activate the project virtual environment:
+
+```
+source ../.venv/bin/activate
+```
+
+Finally, add the `dagster-sling` library to the project. We will also add `duckdb` to use as a destination for our Sling replication.
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/integrations/sling-component/2-add-sling.txt" />
+
+## 2. Scaffold a Sling component
+
+Now that you have a Dagster project, you can scaffold a Sling component:
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/integrations/sling-component/3-scaffold-sling-component.txt" />
+
+The scaffold call will generate a `defs.yaml` file and a unpopulated Sling `replication.yaml` file:
+
+<CliInvocationExample path="docs_snippets/docs_snippets/guides/components/integrations/sling-component/4-tree.txt" />
+
+In its scaffolded form, the `defs.yaml` file contains the configuration for your Sling workspace:
+
+<CodeExample path="docs_snippets/docs_snippets/guides/components/integrations/sling-component/5-component.yaml" title="my_project/defs/sling_ingest/defs.yaml" language="yaml" />
+
+The `replication.yaml` file still needs to be configured:
```

Comment:
I would word this a bit differently, along the lines of "the generated file is empty, and still needs to be configured" (very minor)